pkg/serviceaccount/jwt.go

```diff
@@ -37,8 +38,10 @@ import (
 // ServiceAccountTokenGetter defines functions to retrieve a named service account and secret
 type ServiceAccountTokenGetter interface {
 	GetServiceAccount(namespace, name string) (*v1.ServiceAccount, error)
+	GetServiceAccountWithMultiTenancy(tenant, namespace, name string) (*v1.ServiceAccount, error)
 	GetPod(namespace, name string) (*v1.Pod, error)
```

Comment:
There is no multi-tenancy change for GetPod()? How can we get a pod in a non-default tenant?

Reply:
GetPod() is used in claims.go and is not called in this change. this change is calling the Validate() function in legacy.go.

Reply:
replaced the default tenants in the tests